### ISSUE TITLE
[2][ahdigital] Confusing orderbook orientation arrows

### ISSUE THREAD

**Issue by tbone-bts** [bitshares/bitshares-ui#1138]
The arrows that swap orderbook orientation are very confusing.  Even after many months, I still never really know which arrows I need to click without trial and error pretty much each and every time, and it creates a sense of feeling lost.  What's causing this is that there are multiple sets of these arrows, some do the same thing (like simply swapping left/right or up/down) while others completely change orientation (i.e. vertical vs. horizontal orderbook)...and it gets even more confusing after you've moved something and want to move it back.    

![orderbook_switcher1](https://user-images.githubusercontent.com/15830348/36000717-e169730e-0cf1-11e8-8946-909e9344b4f0.jpg)

This can be fixed very easily by **(a)** matching the existing "Horizontal" button on the vertical orderbook with a "Vertical" button on the horizontal orderbook.  Also, **(b)** I don't think there is a need to left/right swap the horizontal orderbooks independently of the order forms (they can just swap together via one set of arrows).  So you can get rid of the left/right arrows on the orderbook.  This way there's just 1 set of left/right arrows and 1 set of up/down arrows (both on the order form), and a set of horizontal/vertical buttons (on the orderbook).  

This would make it much more clear what does what here, and avoid having the user feeling lost.  

![dex_order_book_orientation_button_1_2](https://user-images.githubusercontent.com/15830348/36000753-f6aaeec8-0cf1-11e8-994b-437df1eba89a.jpg)

![dex_order_book_orientation_button_1_1](https://user-images.githubusercontent.com/15830348/36000781-107807aa-0cf2-11e8-8c45-18f1376a4d81.jpg)

**Comment by wmbutler** [comment]
I'm hoping @ahdigital will take an interest in this.

**Comment by ahdigital** [comment]
I'll have look when I'm on a desktop next.

**Comment by ahdigital** [comment]
I think I've helped ease the pain with this particular issue by making it work like this:

Controls for position/orientation will only be shown in the upper and bottom left quadrants.

To minimize confusion with the vertical/horizontal order book I've replaced the arrows that were there with a pin icon. The idea being that a user pins and unpins the order book. To unpin the orderbook in its vertical position the user can click the unpin icon, which is located in the same place as before for users already used to this pattern.

I feel this will make the controls more predicable and take away some of the guess work being experienced.

**Example**

![order-book-orientation](https://user-images.githubusercontent.com/3088454/36208099-da096130-118f-11e8-97f8-b1fe5fb4b603.gif)
